### ISSUE TITLE
python script plugin did not accept the script

### ISSUE THREAD

**Issue by cedenoaugusto** [emmetio/npp#12]
I got the exception when I pressed: ctrl+alt+enter "python script plugin did not accept the script"

What can I do?

**Comment by geandre** [comment]
Thank you mmosoll, running on Notepad++ Je Suis Charlie Edition...

**Comment by darknawg63** [comment]
Helped me out. Thanks mmosoll!

**Comment by TanmayDharmaraj** [comment]
Worked for me. thanks a lot

**Comment by superadministrator** [comment]
Thank you mmosoll, I tried your solution, it worked for me!

**Comment by pplflymoon** [comment]
I settled this problem as follows:
1.Manual installation Emmet
Manual installation

Install Python Script plugin (available in Plugin Manager).
Download EmmetNPP plugin and unpack it into C:\Program Files\Notepad++\plugins folder
Start Nodepad++. You should see Plugins > Emmet menu item

2.Replace Notepad++ Python package with PythonScript_full_0.9.2.0.7z as mmosoll said before.

3.change the shortcut mapper Settings >  setting-plugin commands --Expand abbreviation to tab

**Comment by cedenoaugusto** [comment]
I followed yours instruccions, but it does not work for me. Instead I have been using Brackets.

Wrote: "table>tr>td"

Then, in the Menu: Plugin > Emmet > Expand...

Produces:
Message1: Unknown exception
Message2: Phyton Script Plugin did not accept the script


Win 7
NP++ v6.7.5

**Comment by pplflymoon** [comment]
hi~~Augusto Cedeño 
1. make sure you Manual installation Emmet
2.down load Python Script Plugin,http://sourceforge.net/projects/npppythonscript/files/Python%20Script%200.9.2.0/
then replace Notepad++ Python 




2015-05-15 

pplflymoon2013 



发件人：Augusto Cedeño <notifications@github.com>
发送时间：2015-05-15 05:00
主题：Re: [npp] python script plugin did not accept the script (#12)
收件人："emmetio/npp"<npp@noreply.github.com>
抄送："pplflymoon"<pplflymoon2013@gmail.com>

I followed yours instruccions, but it does not work for me. Instead I have been using Brackets.
Wrote: "table>tr>td"
Then, in the Menu: Plugin > Emmet > Expand...
Produces:
Message1: Unknown exception
Message2: Phyton Script Plugin did not accept the script
Win 7
NP++ v6.7.5

**Comment by mmosoll** [comment]
The "Phyton Script Plugin did not accept the script"  error appeared with the old phyton scripts distributed with NP++.

The last Notepad++ v6.7.7 with Emmet plugin (last version is 1.0.2) is running fine (only time lag when expresion is too large). I updated NP++ but PythonScripts were updated (several version ago) with the PythonScript_full_0.9.2.0.7z archive:

http://sourceforge.net/projects/npppythonscript/files/Python%20Script%200.9.2.0/

Unpack PythonScript_full_0.9.2.0.7z in NP++ main directory and check also your Shortcut Mapper if necessary.

**Comment by wassem** [comment]
hi
i have notepad++ version 6.8.2
Emmet plugin 1.0.0 
python script 1.0.6.0
Python 2.7.6-notepad++ r2 (default, Apr 21 2014, 19:26:54) [MSC v.1600 32 bit (Intel)]


same problem 
python script plugin did not accept the script 

?

**Comment by mmosoll** [comment]
I am using notepad++ v6.8.2
Emmet plugin 1.0.2 running fine with PythonScript_full_0.9.2.0.7z (see previous post)

**Comment by BugZappr** [comment]
OK, so I inspected the 0.9.2.0 archive for PythonScript, and noticed that it contained \plugins\PythonScript\lib, \plugins\PythonScript\scripts and \docs directories that were not present in my installation in which Emmet was malfunctioning.  Not content to do a needless downgrade, I simply dropped these folders into my Notepad++ dir.  Emmet started working, even with the PythonScript installed through the plugin manager.  Not being content with out of date scripts, I deleted the scripts and docs I had just dropped in, and dropped in the entire contents of the 1.0.8.0 plugin script folder.  It appears to be running perfectly fine now; even picking up a custom snippets.json definition.

It appears that what is now needed to fix this plugin is not for someone to update the Emmet plugin to run with the new PythonScript, which appears to be fully compatible with the Emmet plugin, but that the additional folders and files in the downloadable PythonScript archives (I used "PythonScript_Full_1.0.8.0.zip") also be included in the Plugin Manager's installation (provided it can do it).

Hope this shortens someone's path, and helps ppl find the Zen of Emmet coding sooner.

**Comment by mmosoll** [comment]
Very good job BugZappr,

Summarizing your work:
1. From Notepad++ main directory
- Remove PythonScript.dll
- Remove PythonScript directory
- Remove doc\PythonScript directory

2. I used PythonScript from http://sourceforge.net/projects/npppythonscript/files/Python%20Script%201.0.8.0/PythonScript_Full_1.0.8.0.zip
and copy the following extracted files plugins\PythonScript.dll, plugins\doc\, plugins\PythonScript\ to the Notepad++ main directory.

At first glance, I think with this update Emmet speed performance was increased :-)

Thank you

**Comment by lwd1900** [comment]
hi
i have notepad++ version 6.8.2

Emmet plugin 1.0.0

python script 1.0.6.0

Emmet is no work..........

What can I do? Thanks in advance.

**Comment by cedenoaugusto** [comment]
how to unsuscribed to this list?

Saludos,

ACP
_____________________________________________________________
*Augusto J.A. Cedeño Picón*
7902 8106
*cedeno.augusto@gmail.com <cedeno.augusto@gmail.com>*
Antes de imprimir piense bien si es necesario hacerlo: El Medio Ambiente es
cosa de todos.

**Comment by BugZappr** [comment]
lwd1900: Those are the precise versions I had.  You will need to copy the contents of the above given archive into yer NPP dir; not just install it from the plugin manager.  As this issue is closed, presumably the plugin manager's archive is being updated.

**Comment by mac10046** [comment]
Thanx bro - I am using 6.8.6 version and your solution actually worked .... @mmosoll

**Comment by Mohammad-H** [comment]
Thank you mmosoll

**Comment by Rafiozoo** [comment]
Almost worked on my machine: only choosing from menu Emmet expanded or wrapped, because default shortcuts (`Ctrl-Alt-Enter`, `Ctrl-Shift-Alt-Enter`) are overridden. I didn't searched for conflicts, just set new shortcuts (`Ctrl-Shift-E` for expanding, and `Ctrl-Shift-A` for wrapping work fine).

**Comment by Kaderbhai** [comment]
Thank you mmosoll

**Comment by nicolasfuentes** [comment]
I can't believe, it's working! 
thanks mmosoll and pplflymoon

**Comment by edgexie** [comment]
I can't download the Python at the link that you provide,mmosoll. it's too slow . Wuuuu, I'm in china.

**Comment by mmosoll** [comment]
Try to download it using this link (compressed with 7zip):
https://www.dropbox.com/s/n9t56vnnre27q3j/PythonScript_Full_1.0.8.0.7z

**Comment by cedenoaugusto** [comment]
Now it works.

After I downloaded and installed Python Script Plugin: http://sourceforge.net/projects/npppythonscript/files/Python%20Script%200.9.2.0/

Thanks pplflymoon and mmosoll.

**Comment by fzhw88** [comment]
1.0.8.0 Python Script is OK,but you must installed the full vision by yourself not by plugin Manager

**Comment by BugZappr** [comment]
I've installed it via plugin manager.  I reported the problem with the libraries to the Python plugin guys.  It now works if you just install it via the plugin manager.

**Comment by LaitonGarcia** [comment]
Aqui funfou beleza mmosoll, Obrigado!

**Comment by ybhuxiao** [comment]
Thank you mmosoll, worked for me!

**Comment by RowboTony** [comment]
yep, solution https://github.com/emmetio/npp/issues/12#issuecomment-57280744 still works, thanks!

**Comment by BugZappr** [comment]
I'm glad it works now.  You guys all seem to think the solution was given by mmosoll; but I was the one who first posted it for him to restate, FWIW!  You can read it in the comments above for yourselves.  Emmit now works when downloaded with PythonScript from the Plugin Manager; although NPP's Plugin Manager has an ad graphic, and they have quit using Plugin Manager by default.  They will replace it with a banner-free version soon.  You can still install Plugin Manager as an optional plug-in.  I recently tested this on a PortableApps version starting from a clean install.  NPP's PythonScript now downloads and installs Python's libraries too.

**Comment by R-14** [comment]
Hello, when I installed emmet and notepad++ there were no pre-existing pythonscript.dll files. I went to the link that @BugZappr suggested and downloaded a file from there. I extracted the Pythonscript.dll files to C/ProgrammeFiles/Notepad++/Plugins. However when I open up Notepad++ and I try to install pythonscript via the plugin manager I continue to get an error message saying installation failed. Where am I going wrong? I can install Python Indent but not the script files...
![2018-03-01](https://user-images.githubusercontent.com/19887948/36823832-fe3c6436-1cf6-11e8-898a-dbe65792da61.png)
